easybuild/easyconfigs/s/stardist/stardist-0.8.3-foss-2021b.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'stardist'
+version = '0.8.3'
+
+homepage = 'https://github.com/stardist/stardist'
+description = "Object Detection with Star-convex Shapes."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('TensorFlow', '2.7.1'),
+    ('numba', '0.54.1'),
+    ('imageio', '2.13.5'),
+    ('scikit-image', '0.19.1'),
+    ('tqdm', '4.62.3'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('tifffile', '2022.10.10', {
+        'checksums': ['50b61ba943b866d191295bc38a00191c9fdab23ece063544c7f1a264e3f6aa8e'],
+    }),
+    ('csbdeep', '0.7.2', {
+        'checksums': ['8a000eb71d04aa753f52ffe81a34c0a30a06ee986d20b9986d76272253e2fd53'],
+    }),
+    ('edt', '2.3.0', {
+        'checksums': ['220e2086fdf32bbd8964df76b3e1a42061ece1f4b9ea95ec83a094d24d258664'],
+    }),
+    (name, version, {
+        'checksums': ['8ac1f6165ca8e8496651c6e2b49ce57d6d574dfb22bf6f63ddaec1f64b8868d1'],
+    }),
+]
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/stardist-predict2d'],
+    'dirs': ['bin'],
+}
+
+moduleclass = 'tools'
```

Comment:
Maybe `vis` is better, since it's a tool for detection of patterns in images?
Not sure though, `tools` seems perhaps too generic...

Reply:
`tools` is indeed too generic. Probably `vis` is the best of the available choices.